app/view/twig/editcontent/fields/_relationship.twig

```diff
@@ -6,10 +6,12 @@
 {# Use option groups #}
 {% set groupby = relation.groupby|default(false) %}
 
+{% set values = context.content.relation.getField(relcontenttype, true, context.contenttype.slug, context.content.id)|default([]) %}
+
 {# Currently selected relationship values #}
 {% set relselect = [] %}
-{% for item in context.content.relation.getField(relcontenttype, true)|default([]) %}
-    {% set relselect =  relselect|merge([item.to_id]) %}
+{% for item in values %}
+    {% set relselect =  relselect|merge([item.toid]) %}
 {% endfor %}
```

Comment:
OK… now going into COD mode … and would just ninja this but figure it's better to make it part of the dialogue.

We probably should put a `{{# comment #}}` explaining the "why" one vs the other … will happily do if your dance card is full though :+1: